static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.spec.tsx

```diff
@@ -16,7 +16,7 @@ describe('TeamIssuesBreakdown', () => {
       url: `/teams/${organization.slug}/${team.slug}/issue-breakdown/`,
       body: TeamIssuesBreakdownFixture(),
     });
-    const statuses = ['new', 'regressed', 'unignored'];
+    const statuses = ['new', 'regressed', 'unarchived'];
```

Comment:
do we support unarchived as a status? 🤔

Reply:
idk 😓 what's the new version of unignored? escalating?

Reply:
ah yeah, escalating is probably more accurate!

Reply:
hmm now I'm confused - I'm getting this typing error:

```
Error: static/app/views/organizationStats/teamInsights/issues.tsx(111,48): error TS2322: Type '"escalating"' is not assignable to type '"new" | "deleted" | "ignored" | "resolved" | "regressed" | "archived" | "unarchived" | "unignored"'.
```

and it looks like `unarchived` is defined [here](https://github.com/getsentry/sentry/blob/master/static/app/views/organizationStats/teamInsights/teamIssuesBreakdown.tsx#L21-L31) but `escalating` is not - I don't have any context around this, do you have any idea what's correct?

Reply:
hm idk since this is some stats breakdown - "unarchive" is probably correct there then!